VERSION ""


NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    BA_DEF_DEF_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_SGTYPE_
    BA_SGTYPE_
    SIG_TYPE_REF_
    VAL_TABLE_
    SIG_GROUP_
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    BO_TX_BU_
    BA_DEF_REL_
    BA_REL_
    BA_DEF_DEF_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_:

BU_: XXX


BO_ 2 Steering: 8 XXX
 SG_ Counter : 25|3@1+ (1,0) [0|7] "" XXX
 SG_ Checksum : 39|8@0+ (1,0) [0|255] "" XXX
 SG_ Steering_angle : 7|16@0- (1,0) [0|65535] "" XXX

BO_ 64 Accelerator: 8 XXX
 SG_ NEW_SIGNAL_1 : 32|8@1+ (1,0) [0|255] "" XXX
 SG_ Counter : 11|4@0+ (1,0) [0|255] "" XXX
 SG_ Checksum : 7|8@0- (1,0) [0|255] "" XXX
 SG_ Off_Accel : 63|4@0+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_3 : 56|4@1+ (1,0) [0|255] "" XXX
 SG_ Accel_Pedal : 55|8@0+ (1,0) [0|255] "" XXX
 SG_ Engine_RPM : 16|12@1+ (1,0) [0|255] "" XXX

BO_ 65 NEW_MSG_1: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 32|12@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 16|12@1+ (1,0) [0|1] "" XXX
 SG_ NEW_SIGNAL_5 : 31|1@0+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_6 : 48|8@1+ (1,0) [0|63] "" XXX
 SG_ NEW_SIGNAL_7 : 59|2@0+ (1,0) [0|255] "" XXX

BO_ 72 NEW_MSG_2: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 40|16@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 38|3@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_5 : 16|8@1+ (1,0) [0|255] "" XXX

BO_ 316 NEW_MSG_3: 8 XXX

BO_ 326 NEW_MSG_4: 8 XXX
 SG_ NEW_SIGNAL_1 : 16|12@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 32|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 40|2@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_5 : 7|8@0+ (1,0) [0|255] "" XXX

BO_ 315 G_Sensor: 8 XXX
 SG_ longitudinal : 63|8@0- (1,0) [0|255] "" XXX
 SG_ Latitudinal : 48|8@1- (1,0) [0|255] "" XXX

BO_ 314 Wheel_Speed: 8 XXX
 SG_ NEW_SIGNAL_4 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_6 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_5 : 12|13@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_2 : 25|13@1+ (1,0) [0|65535] "" XXX
 SG_ NEW_SIGNAL_3 : 51|13@1+ (1,0) [0|32767] "" XXX
 SG_ NEW_SIGNAL_1 : 38|13@1+ (1,0) [0|63] "" XXX

BO_ 73 NEW_MSG_5: 8 XXX
 SG_ NEW_SIGNAL_1 : 39|8@0+ (1,0) [0|4095] "" XXX
 SG_ NEW_SIGNAL_2 : 24|8@1+ (1,0) [0|127] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX

BO_ 280 NEW_MSG_6: 8 XXX
 SG_ NEW_SIGNAL_1 : 12|12@1- (1,0) [0|4095] "" XXX
 SG_ NEW_SIGNAL_2 : 48|8@1- (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 61|1@0+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_4 : 40|4@1+ (1,0) [0|255] "" XXX

BO_ 281 NEW_MSG_7: 8 XXX
 SG_ NEW_SIGNAL_2 : 56|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_1 : 55|8@0- (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 40|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_5 : 39|8@0+ (1,0) [0|127] "" XXX

BO_ 312 Brake_Pressure_L_R: 8 XXX
 SG_ NEW_SIGNAL_2 : 48|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 56|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_1 : 16|9@1+ (1,0) [0|255] "" XXX

BO_ 313 NEW_MSG_8: 8 XXX
 SG_ NEW_SIGNAL_2 : 48|16@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_1 : 16|16@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 32|16@1+ (1,0) [0|65535] "" XXX
 SG_ NEW_SIGNAL_4 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_5 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 290 ES_NEW_MSG_9: 8 XXX
 SG_ Checksum : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 12|1@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_1 : 23|16@0+ (1,0) [0|3] "" XXX

BO_ 722 NEW_MSG_10: 8 XXX
 SG_ NEW_SIGNAL_1 : 27|3@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 56|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_5 : 45|2@0+ (1,0) [0|3] "" XXX

BO_ 544 ES_NEW_MSG_11: 8 XXX
 SG_ NEW_SIGNAL_3 : 16|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 36|4@1+ (1,0) [0|63] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 545 ES_Distance: 8 XXX
 SG_ Close_Distance : 47|8@0+ (1,0) [0|255] "" XXX
 SG_ Distance_Swap : 37|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_5 : 39|1@0+ (1,0) [0|31] "" XXX
 SG_ NEW_SIGNAL_9 : 38|1@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_4 : 34|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_8 : 36|1@0+ (1,0) [0|3] "" XXX
 SG_ Brake_Pedal : 33|1@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_6 : 32|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_10 : 35|1@1+ (1,0) [0|3] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ Throttle : 16|12@1+ (1,0) [0|65535] "" XXX
 SG_ NEW_SIGNAL_1 : 31|4@0+ (1,0) [0|15] "" XXX

BO_ 546 ES_NEW_MSG_12: 8 XXX
 SG_ Brake_Pedal : 30|1@0+ (1,0) [0|3] "" XXX
 SG_ RPM : 16|12@1+ (1,0) [0|255] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_1 : 29|1@0+ (1,0) [0|3] "" XXX

BO_ 554 ES_NEW_MSG_13: 8 XXX
 SG_ NEW_SIGNAL_3 : 39|16@1+ (1,0) [0|16777215] "" XXX
 SG_ NEW_SIGNAL_4 : 13|1@0+ (1,0) [0|3] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 557 NEW_MSG_14: 8 XXX
 SG_ NEW_SIGNAL_1 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_2 : 7|8@0+ (1,0) [0|255] "" XXX

BO_ 576 NEW_MSG_15: 8 XXX
 SG_ NEW_SIGNAL_1 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_2 : 7|8@0+ (1,0) [0|255] "" XXX

BO_ 577 NEW_MSG_16: 8 XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_1 : 16|12@1+ (1,0) [0|255] "" XXX

BO_ 552 NEW_MSG_17: 8 XXX
 SG_ NEW_SIGNAL_1 : 48|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 7|8@0+ (1,0) [0|255] "" XXX

BO_ 912 NEW_MSG_18: 8 XXX
 SG_ NEW_SIGNAL_1 : 32|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 50|1@0+ (1,0) [0|3] "" XXX

BO_ 940 Door: 8 XXX
 SG_ NEW_SIGNAL_1 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_2 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 54|1@0+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_4 : 32|1@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_5 : 33|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_6 : 39|1@0+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_7 : 35|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_8 : 16|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_9 : 31|3@0+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_10 : 13|1@0+ (1,0) [0|3] "" XXX

BO_ 801 ES_NEW_MSG_20: 8 XXX
 SG_ NEW_SIGNAL_5 : 52|1@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_6 : 36|2@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_7 : 42|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_8 : 41|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_9 : 63|4@0+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_10 : 49|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_11 : 45|2@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_12 : 29|1@0+ (1,0) [0|7] "" XXX
 SG_ Brake_Pedal : 51|1@0+ (1,0) [0|3] "" XXX
 SG_ Distance : 56|4@1+ (1,0) [0|15] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_1 : 28|1@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_2 : 40|1@1+ (1,0) [0|3] "" XXX

BO_ 802 ES_NEW_MSG_21: 8 XXX
 SG_ NEW_SIGNAL_2 : 32|5@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_4 : 44|1@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_5 : 56|2@1+ (1,0) [0|3] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_1 : 17|2@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_3 : 26|1@1+ (1,0) [0|7] "" XXX
 SG_ NEW_SIGNAL_6 : 24|1@1+ (1,0) [0|3] "" XXX

BO_ 805 ES_NEW_MSG_22: 8 XXX
 SG_ NEW_SIGNAL_3 : 23|2@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_4 : 15|1@0+ (1,0) [0|3] "" XXX
 SG_ Checksum : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_1 : 14|1@0+ (1,0) [0|3] "" XXX

BO_ 808 NEW_MSG_23: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 837 NEW_MSG_24: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 40|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 32|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_5 : 24|8@1+ (1,0) [0|255] "" XXX

BO_ 838 NEW_MSG_25: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 16|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 40|1@1+ (1,0) [0|3] "" XXX

BO_ 842 NEW_MSG_26: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 32|8@1+ (1,0) [0|255] "" XXX

BO_ 915 NEW_MSG_27: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 16|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 32|9@1+ (1,0) [0|255] "" XXX

BO_ 1788 NEW_MSG_28: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 40|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_4 : 48|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_5 : 16|8@1+ (1,0) [0|255] "" XXX

BO_ 816 NEW_MSG_29: 8 XXX
 SG_ NEW_SIGNAL_1 : 7|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 826 NEW_MSG_30: 8 XXX
 SG_ NEW_SIGNAL_1 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_2 : 0|8@1+ (1,0) [0|255] "" XXX

BO_ 839 NEW_MSG_31: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 2015 NEW_MSG_32: 8 XXX
 SG_ NEW_SIGNAL_2 : 16|8@1+ (1,0) [0|255] "" XXX

BO_ 2024 NEW_MSG_33: 8 XXX
 SG_ NEW_SIGNAL_1 : 31|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 0|3@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 16|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_4 : 32|8@1+ (1,0) [0|255] "" XXX

BO_ 1614 NEW_MSG_34: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 1617 NEW_MSG_35: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 1632 NEW_MSG_36: 8 XXX
 SG_ NEW_SIGNAL_1 : 55|16@0+ (1,0) [0|255] "" XXX

BO_ 1650 NEW_MSG_37: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 1657 NEW_MSG_38: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 1658 NEW_MSG_39: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 33|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_4 : 31|1@0+ (1,0) [0|3] "" XXX

BO_ 1677 NEW_MSG_40: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_3 : 16|4@1+ (1,0) [0|15] "" XXX

BO_ 1743 NEW_MSG_41: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|8@1+ (1,0) [0|127] "" XXX
 SG_ NEW_SIGNAL_2 : 8|4@1+ (1,0) [0|15] "" XXX

BO_ 1785 NEW_MSG_42: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|4@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 21|1@1+ (1,0) [0|31] "" XXX
 SG_ NEW_SIGNAL_3 : 17|1@1+ (1,0) [0|7] "" XXX

BO_ 1759 NEW_MSG_43: 8 XXX
 SG_ NEW_SIGNAL_1 : 17|1@1+ (1,0) [0|3] "" XXX

BO_ 1786 NEW_MSG_44: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_2 : 16|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 8|8@1+ (1,0) [0|15] "" XXX

BO_ 1787 NEW_MSG_45: 8 XXX
 SG_ NEW_SIGNAL_1 : 0|4@1+ (1,0) [0|15] "" XXX
 SG_ NEW_SIGNAL_2 : 8|8@1+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 16|8@1+ (1,0) [0|255] "" XXX
